### PR TITLE
Add custom sorter functionality to multiple sort

### LINKED ISSUE #3680: Can the multiple-sort extension use sorter functions, similar to the single-column sort?
When sorting on single columns, you can specify a sorter function with the data-sorter tag.  We use this to sort date strings chronologically rather than alphabetically.

But it doesn't look like the multiple-sort extension uses the data-sorter tag, as any multiple sort using a field with my date sorter function still sorts the field alphabetically.

Is there a way to specify a sorter function with multiple sort?

If not ... that'd be a great addition.

I'm not sure I could quickly figure this out on my own, but if you can point me in the right direction, I'm willing to give it a try. Would appreciate your advice ... thanks!

### PR DESCRIPTION
**Pull request by jbarrineau06**
Added functionality to check for and use a custom sorter if it exists on the column for the multiple-sort extension.

Before: https://live.bootstrap-table.com/code/jbarrineau06/1749
After: https://live.bootstrap-table.com/code/jbarrineau06/1750

fix #3680

### CODE REVIEW

**Comment by UtechtDustin** [comment]
Could you please provide us an example of that PR using our [editor](https://live.bootstrap-table.com/)?
You can use your branch by selecting `From GitHub src` and paste your branch `jbarrineau06:add-custom-sorter-functionality-to-multiple-sort` in the `GitHub src branch` input.

**Comment by jbarrineau06** [comment]
I added examples of the behavior before and after the change.

**Comment by UtechtDustin** [comment]
https://github.com/wenzhixin/bootstrap-table/pull/4855/files#diff-269b598a799964a4d52ac38fa4362a1bR602-R611
Please remove the comments (i think we dont need them) and the empty lines between the variables.
But for me it looks good, thank you.

@wenzhixin @djhvscf 
Could somebody of you give a second review for that PR ?

File: src/extensions/multiple-sort/bootstrap-table-multiple-sort.js [hunk 1]
```diff
@@ -599,18 +599,36 @@ BootstrapTable.prototype.onMultipleSort = function () {
     const arr2 = []
 
     for (let i = 0; i < that.options.sortPriority.length; i++) {
-      const order = that.options.sortPriority[i].sortOrder === 'desc' ? -1 : 1
-      let aa = a[that.options.sortPriority[i].sortName]
-      let bb = b[that.options.sortPriority[i].sortName]
+      // get fieldName from multi sort
```
Comment (UtechtDustin) [inline src/extensions/multiple-sort/bootstrap-table-multiple-sort.js]: remove comments and remove the empty lines between the variables

**Comment by UtechtDustin** [comment]
@jbarrineau06 
After the change request is done, we can merge that!

File: src/extensions/multiple-sort/bootstrap-table-multiple-sort.js (hunk 1 above)
Comment (jbarrineau06) [reply src/extensions/multiple-sort/bootstrap-table-multiple-sort.js]: Removed comments and empty lines in latest commit

**Comment by UtechtDustin** [comment]
Thank you for your contribution! :-)

**Comment by jbarrineau06** [comment]
No problem!

**Pull request merged**